ouroboros-consensus/docs/interface-CHANGELOG.md

```diff
@@ -56,6 +56,15 @@ may appear out of chronological order.
 The internals of each entry are organized similar to
 https://keepachangelog.com/en/1.1.0/, adapted to our plan explained above.
 
+## Circa 2022-10-07
```

Comment:
We can update the date on this entry.

And also mention `added Conway to CardanoEras`. Hmm, that change blurs the line between `Added` and `Changed`, doesn't it. I suppose it'd be a `Changed`. Maybe we could also have an `Added` entry just for adding all the Conway stuff (eg newer `*VersionNN*` patterns, Conway type synonyms, etc etc).

Reply:
I added an `Added` entry :) I'd need your help for completing the other "Conway stuff' part.

Reply:
Mention addition of NTN v1 and NTC v15, which both enable Conway awareness.

Mention that we now export `*Conway*` type and pattern synonyms. EG `StandardConway`, `BlockConway`, `LedgerErrorConway`, etc, etc.

Reply:
Done. For the last item, I did not list everything. I don't think it's necessary.